imgix/urlbuilder.py

```diff
@@ -322,12 +328,23 @@ def _build_srcset_DPR(
         srcset_params = dict(params)
         srcset_entries = []
 
+        variable_qualities = options.get("variable_qualities", {})
+        if variable_qualities:
+            validate_variable_qualities(variable_qualities)
+
+        qualities = {**DPR_QUALITIES, **variable_qualities}
+
         for dpr in targets:
             srcset_params["dpr"] = dpr
 
             if not disable_variable_quality:
-                quality = params.get("q", DPR_QUALITIES[dpr])
-                srcset_params["q"] = quality
+                quality = (
+                    params.get("q", None)
+                    or qualities.get("dpr")
```

Comment:
Could you elaborate on the purpose of checking for this? I tried testing the following and there appears to have no effect:

```python
client.create_srcset("image.jpg", {"w": 100, "dpr":1}, {"variable_qualities": {1: 100, 2: 90, 3: 80}})
```

Reply:
Ok that makes sense. We'll have to check back in on this specific condition another time.